app/models/concerns/user_receivable.rb

```diff
@@ -18,7 +18,33 @@ def unreceive(channel)
 
       if reception.present?
         reception.destroy
-        ChannelWorksDestroyingWorker.perform_async(id, channel.id)
+        delay.destroy_channel_works(self, channel)
+      end
+    end
+
+    private
+
+    def create_channel_works(user, channel)
```

Comment:
Assignment Branch Condition size for create_channel_works is too high. [16.79/15]